lrrbot/commands/static.py

```diff
@@ -1,86 +1,143 @@
-			fragment += "Throttled: 30\n"
-			fragment += "Throttle-Count: 2\n"
-			fragment += "Literal-Response: true\n"
-			if access == "sub":
-				fragment += "Sub-Only: true\n"
-			elif access == "mod":
-				fragment += "Mod-Only: true\n"
-			fragment += "Section: text\n"
-			fragment += "\n"
-			response = response if isinstance(response, str) else response[0]
-			fragment += response + "\n"
-			yield fragment
-	return "\n--command\n".join(generator())
-
-def generate_expression():
-	return "(%s)" % "|".join(re.escape(c).replace("\\ ", " ") for c in storage.data["responses"])
+		with bot.engine.connect() as conn:
+			query = sqlalchemy.select(commands.c.id, commands.c.access, aliases.c.alias)
+			query = query.select_from(commands.join(aliases, aliases.c.command_id == commands.c.id))
+			query = query.order_by(commands.c.id, aliases.c.id)
+
+			first = True
+			for command_id, rows in itertools.groupby(conn.execute(query), key=lambda row:row[0]):
+				if not first:
+					yield "--command"
+				first = False
+
+				for _, access, alias in rows:
+					yield f"Command: {config['commandprefix']}{alias}"
+				yield "Throttled: 30"
+				yield "Throttle-Count: 2"
+				yield "Literal-Response: true"
+				if access == ACCESS_SUB:
+					yield "Sub-Only: true"
+				elif access == ACCESS_MOD:
+					yield "Mod-Only: true"
+				yield "Section: text"
+				yield ""
+
+				resp_query = sqlalchemy.select(responses.c.response).where(responses.c.command_id == command_id)
+				resp_query = resp_query.order_by(responses.c.id).limit(1)
+				response, = conn.execute(resp_query).first()
+				yield response
+
+	return "\n".join(generator())
+
+def generate_expression(bot):
+	aliases = bot.metadata.tables["commands_aliases"]
+	with bot.engine.connect() as conn:
+		query = sqlalchemy.select(aliases.c.alias)
+		aliases = conn.execute(query)
+		aliases = [alias for alias, in aliases]
+	return "(%s)" % "|".join(re.escape(c).replace("\\ ", " ") for c in aliases)
 
 @lrrbot.decorators.throttle(30, params=[4], count=2)
 def static_response(bot, conn, event, respond_to, command):
-	command = " ".join(command.split())
-	response_data = storage.data["responses"][command.lower()]
-	source = irc.client.NickMask(event.source)
-	if response_data["access"] == "sub":
-		if not bot.is_sub(event) and not bot.is_mod(event):
-			log.info("Refusing %s due to inadequate access" % command)
-			conn.privmsg(source.nick, "That is a sub-only command.")
+	commands = bot.metadata.tables["commands"]
+
+	with bot.engine.connect() as dbconn:
+		command = " ".join(command.split())
+		command_id = _get_command_id(bot, dbconn, command)
+		if command_id is None:
 			return
-	if response_data["access"] == "mod":
-		if not bot.is_mod(event):
-			log.info("Refusing %s due to inadequate access" % command)
-			conn.privmsg(source.nick, "That is a mod-only command.")
+
+		query = sqlalchemy.select(commands.c.access).where(commands.c.id == command_id)
+		access, = dbconn.execute(query).first()
```

Comment:
This is also [`Result.scalar()`](https://docs.sqlalchemy.org/en/20/core/connections.html#sqlalchemy.engine.Result.scalar).
```suggestion
		access = dbconn.execute(query).scalar()
```